manifest.xml

```diff
@@ -38,9 +38,9 @@
 		</Hosts>
 		<Resources>
 			<bt:Urls>
-				<bt:Url id="JSON-URL" DefaultValue="https://localhost:3000/config/customfunctions.json" />
-				<bt:Url id="JS-URL" DefaultValue="https://localhost:3000/src/customfunctions.js" />
-				<bt:Url id="HTML-URL" DefaultValue="https://localhost:3000/index.html" />
+				<bt:Url id="JSON-URL" DefaultValue="https://localhost:8081/config/customfunctions.json" />
+				<bt:Url id="JS-URL" DefaultValue="https://localhost:8081/dist/win32/ship/index.win32.bundle" />
```

Comment:
As we've discussed, I believe this should be `https://localhost:8081/index.win32.bundle`. We need to sort this out.

Reply:
Yes, we should get someone else to test out the current manifest and see if it works, because I am not having any luck when the bundle url is https://localhost:8081/index.win32.bundle